backend/spec/model_spreadsheet_builder_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
```

Comment:
@stitching-coder sounds good, we could merge this in the meantime.

We have a new CI workflow, could you rebase this branch on master so that we can run it here as well?